package.json

```diff
@@ -78,7 +78,7 @@
     "@storybook/client-api": ">=6.4.0",
     "@storybook/preview-web": ">=6.4.0",
     "@storybook/react": ">=6.4.0",
-    "react": "^16.8.0 || ^17.0.0"
```

Comment:
Yeah this looks like the proper fix to me.

I'd expect a lock-file change associated with this, @yannbf
if for testing purposes you want to stay on a particular version in this repository (whilst support any listed version) it's a good idea to add the exact version you know this repo is ran with as a devDependency too.

Reply:
@ndelangen 👋 Just in case, I just ran `yarn install` locally and I didn't get any changes on my `yarn.lock` file.